cookbooks/bcpc/recipes/powerdns.rb

```diff
@@ -45,259 +44,214 @@
 subnet = node[:bcpc][:management][:subnet]
 raise "Did not get a subnet" if not subnet
 
-%w{pdns-server pdns-backend-mysql}.each do |pkg|
-    package pkg do
-        action :upgrade
-    end
-end
+if node[:bcpc][:management][:vip] and get_nodes_for("mysql").length() > 0
+#  make_config('mysql-pdns-user', "pdns")
+#  make_config('mysql-pdns-password', secure_password)
+
+  allnodes = Array.new
```

Comment:
Why not just use [`get_node_attributes`](https://github.com/bloomberg/chef-bach/blob/d61cd50e6683474eded3e9fb3007ff370b62c31a/cookbooks/bcpc/libraries/utils.rb#L209-L213)